### PR TITLE
Update OMERO.tables 5.1 docs

### PR DESCRIPTION
**Pull request by manics**
Documentation for openmicroscopy/openmicroscopy#3249 (includes breaking changes)

--no-rebase

### CODE REVIEW

File: omero/developers/whatsnew.txt [hunk 1]
```diff
@@ -11,6 +11,9 @@ write about.
 Breaking changes
 ----------------
 
+Graphs
+~~~~~~
```
Comment (hflynn) [inline omero/developers/whatsnew.txt]: This underlining should be '^^^^' rather than '~~~' - same below.

File: omero/developers/Tables.txt [hunk 2]
```diff
@@ -231,6 +237,32 @@ Main methods
...
+        :param string/int/float/long value: The value of the property.
+
+    .. method:: setAllMetadata(keyvalues)
+
+        Store multiple additional properties associated with a Table. See
+        :meth:`setMetadata()`.
+
+        :param dict keyvalues: A dictionary of key-value pairs.
+
+    .. method:: getMetadata()
```
Comment (rleigh-dundee) [inline omero/developers/Tables.txt]: No key name as an argument?

**Comment by rleigh-dundee** [comment]
Looks fine except for the noted discrepancy where it looks like the keyname would be needed to get a specific property?

**Comment by manics** [comment]
Fixed.

**Comment by hflynn** [comment]
Happy with this now @rleigh-dundee ?

**Comment by rleigh-dundee** [comment]
@hflynn Yes, looks fine, thanks.  Good to merge.

**Pull request merged**